docs/modules/clusters/pages/network-configuration.adoc

```diff
@@ -1021,7 +1021,7 @@ You can specify TCP keep-alive options for each socket for use cases such as pre
 
 These options:
 
-* work with JDK 11 or newer, and recent JDK 8 distributions based on OpenJDK
+* work with JDK 17 or newer, and JDK 17 distributions based on OpenJDK
```

Comment:
I'm not sure this makes sense any more - originally it was saying there were non-OpenJDK, JDK8 distributions where these options won't work.
But if they all work on JDK11, and the minimum supported is 17, then I think these lines can just go?